src/rgw/rgw_lc.cc

```diff
@@ -30,11 +30,9 @@ bool LCRule::valid()
 {
   if (id.length() > MAX_ID_LEN) {
     return false;
-  }
-  else if(expiration.empty() && noncur_expiration.empty() && mp_expiration.empty() && !dm_expiration) {
+  } else if (expiration.empty() && noncur_expiration.empty() && mp_expiration.empty() && !dm_expiration) {
```

Comment:
if we're going to do a spacing cleanup, it would be nice to break >80 column expressions